BTSerial/MainPage.xaml

```diff
@@ -0,0 +1,25 @@
+﻿<Page
+    x:Class="BTSerial.MainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:BTSerial"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+        <StackPanel>
+            <TextBlock x:Name="txtblock1" Text="Step1: Connect BT Device" Margin="10,10,140,0"/>
+            <Button x:Name="ConnectBT" Click="ConnectDevice_Click" Content="Connect" Margin="10,0,0,10" />
+
+            <TextBlock x:Name="txtblock2" Text="Step2: At below please write the message you are going to send" Margin="10,10,0,0"/>
```

Comment:
instead of "at below please write the message...", try "Write the message you are going to send"